Source/WebKit/WebProcess/WebCoreSupport/WebFileSystemStorageConnection.cpp

```diff
@@ -156,7 +156,7 @@ void WebFileSystemStorageConnection::createSyncAccessHandle(WebCore::FileSystemH
 void WebFileSystemStorageConnection::closeSyncAccessHandle(WebCore::FileSystemHandleIdentifier identifier, WebCore::FileSystemSyncAccessHandleIdentifier accessHandleIdentifier, FileSystemStorageConnection::VoidCallback&& completionHandler)
 {
     if (!m_connection)
-        return;
+        return completionHandler(WebCore::Exception { WebCore::UnknownError, "Connection is lost"_s });
```

Comment:
Why do we return an exception here? Can we change completionHandler to be just void()?

Reply:
I am just following the pattern for other methods (when connection is lost, throwing exception). It might be Okay to return null here since if the connection is lost then the handle must be closed in network process.